docs/api/field_type_form_and_template.md

```diff
@@ -17,39 +17,17 @@ The `FieldValueFormMapperInterface::mapFieldValueForm` method accepts two argume
 - `FormInterface` — form for the current Field
 - `FieldData` — underlying data for current field form
 
-You have to add your form type to the content editing form. The example shows how `ezboolean` injects the form:
+You have to add your form type to the content editing form, for example:
 
 ``` php
-use EzSystems\EzPlatformContentForms\Data\Content\FieldData;
-use EzSystems\RepositoryForms\Form\Type\FieldType\CheckboxFieldType;
-use Symfony\Component\Form\FormInterface;
-
-public function mapFieldValueForm(FormInterface $fieldForm, FieldData $data)
-{
-    $fieldDefinition = $data->fieldDefinition;
-    $formConfig = $fieldForm->getConfig();
-
-    $fieldForm
-        ->add(
-            $formConfig->getFormFactory()->createBuilder()
-                ->create(
-                    'value',
-                    CheckboxFieldType::class,
-                    [
-                        'required' => $fieldDefinition->isRequired,
-                        'label' => $fieldDefinition->getName(
-                            $formConfig->getOption('languageCode')
-                        ),
-                    ]
-                )
-                ->setAutoInitialize(false)
-                ->getForm()
-        );
-}
+[[= include_file('code_samples/field_types/2dpoint_ft/src/FieldType/Point2D/Type.php', 6, 7) =]][[= include_file('code_samples/field_types/2dpoint_ft/src/FieldType/Point2D/Type.php', 10, 11) =]][[= include_file('code_samples/field_types/2dpoint_ft/src/FieldType/Point2D/Type.php', 12, 13) =]]
+// ...
+
+[[= include_file('code_samples/field_types/2dpoint_ft/src/FieldType/Point2D/Type.php', 33, 41) =]]
 ```
 
 Your type has to be called `value`.
-In the example above, `CheckboxFieldType::class` is used, but you can use standard Symfony form type instead.
+In the example above, the custom `Point2DType::class` is used, but you can use standard Symfony form type instead.
```

Comment:
```suggestion
In the example above, the custom `Point2DType::class` is used, but you can use a standard Symfony form type instead.
```
Unless there is only one (seems like not), in wchich case the article would be "the".